hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileUtil.java

```diff
@@ -36,8 +36,11 @@
 import java.nio.charset.CharsetEncoder;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.AccessDeniedException;
+import java.nio.file.attribute.PosixFilePermission;
 import java.nio.file.FileSystems;
 import java.nio.file.Files;
+
+import java.util.*;
```

Comment:
no .* expansions, just add any new imports into the existing ones. and tell your IDE to stop combining imports into .* for future hadoop prs. please: imports are backport pain

Reply:
Thank you for your review. It is true that there was a problem when IDE was introduced.